src/sass/pages/ico_landing_page.scss

```diff
@@ -535,10 +535,160 @@ input[type='text']:focus, input[type='submit']:focus {
     }
 }
 
-#ico_subscribe_section, #lykke-section {
+#ico_subscribe_section, #lykke-section, #media-coverage {
     padding: 50px 15px;
 
-    .margin-top-40 {
-        margin-top: 40px;
+    &#ico_subscribe_section {
```

Comment:
this is the parent id (repeated twice, `#ico_subscribe_section, [...] &#ico_subscribe_section` are we repeating the same id? if yes we should change it, if no we should fix this hirearchy